docs/introduction/Examples.md

```diff
@@ -1 +1,137 @@
-## Examples
\ No newline at end of file
+## Examples
+### Simple - reading from an existing xlsx spread sheet.
+The following C plus plus code will read the values from an xlsx file and print the string values to the screen. This is a very simple example to get you started.
+
+```
+#include <iostream>
+#include <xlnt/xlnt.hpp>
+
+using namespace std;
+using namespace xlnt;
+int main()
+{
+    workbook wb;
+    wb.load("/home/timothymccallum/test.xlsx");
+    auto ws = wb.active_sheet();
+    clog << "Processing spread sheet" << endl;
+    for (auto row : ws.rows(false)) 
+	{ 
+	    for (auto cell : row) 
+	        { 
+		    clog << cell.to_string() << endl;
+		}
+	}
+    clog << "Processing complete" << endl;
+    return 0;
+}
+```
+Save the contents of the above file 
+```
+/home/timothymccallum/process.cpp
+```
+Compile by typing the following command
+```
+g++ -std=c++14 -Ixlnt/include -Lxlnt/lib -lxlnt process.cpp -o process
```

Comment:
Did you create an xlnt/lib directory and copy the xlnt library into it? It works for demonstration purposes, but it might be good to make sure this command matches how xlnt would be linked after being build/installed using CMake or else update the CMake script to place xlnt in a lib directory like this.